custom_components/test/fuzzing/fuzzer_utils/ValuePoolFuzzer.py

```diff
@@ -17,19 +131,22 @@ def __int__(self):
     def fuzz(
         self, param_nr: int = 1, types: list = ["INT"], param_combi: int = 1
     ) -> list:
```

Comment:
I don't think the `fuzz()` works as specialised! No test case is working. The function should return a list of list so that the `ParamRunner` can use it.
```python
"""
...
:return: The value pool for fuzzing.
:rtype: list of lists
"""
```
Compare the log file: [fuzzing.log](https://github.com/ThorbenCarl/PyLoxone/files/15501640/fuzzing.log), only one large field with all values is transferred.
```log
2024-05-30 16:40:10 ERROR ParamRunner.py:run Test failed with parameters: [-4.253529586511731e+37, -4.253529586511731e+37, -4.253529586511731e+37, -4.253529586511731e+37, -4.253529586511731e+37, -4.253529586511731e+37, -4.253529586511731e+37, -4.253529586511731e+37, -4.253529586511731e+37,
...
Exception: map_range() takes 5 positional arguments but 729 were given
```